src/sentry/workflow_engine/service/action/impl.py

```diff
@@ -26,3 +27,31 @@ def update_action_status_for_organization_integration(
             integration_id=integration_id,
             dataconditiongroupaction__condition_group__organization_id=organization_id,
         ).update(status=status)
+
+    def update_action_status_for_sentry_app(
+        self,
+        *,
+        organization_id: int,
+        status: int,
```

Comment:
Are we planning to use this to both disable/re-enable sentry app installs in the future?

Reply:
I'm not confident that's possible in the case of only having the sentry app installation uuid because once the uuid disappears it's gone forever.

I was thinking it would be useful to keep the disabled actions around in case you reinstall again and can mass edit all the disabled actions connected to the old uuid